libraries/AP_Filesystem/AP_Filesystem_Sys.cpp

```diff
@@ -246,6 +221,8 @@ int AP_Filesystem_Sys::stat(const char *pathname, struct stat *stbuf)
         errno = ENOENT;
         return -1;
     }
-    stbuf->st_size = sysfs_file_list[pos].filesize;
+    // give a fixed size of 1024 for stat. It is too expensive to
```

Comment:
That's not the constant being used.
```suggestion
    // give a fixed size for stat. It is too expensive to
```